src/kemal/helpers.cr

```diff
@@ -20,6 +20,10 @@ macro return_with(env, status_code = 200, response = "")
   next
 end
 
+macro headers(env, additional_headers)
```

Comment:
Looks like you're right. I tried a method first and it didn't work so I used a macro which did work. I must have been doing something wrong because a method works perfectly fine now. Updated the PR, thanks for the input.